source/NPC.cpp

```diff
@@ -444,10 +451,15 @@ void NPC::Do(const ShipEvent &event, PlayerInfo &player, UI *ui, bool isVisible)
 				| ShipEvent::BOARD | ShipEvent::CAPTURE | ShipEvent::PROVOKE);
 
 	// Apply this event to the ship and any ships it is carrying.
-	actions[ship.get()] |= type;
+	shipActions[ship.get()] |= type;
 	for(const Ship::Bay &bay : ship->Bays())
 		if(bay.ship)
-			actions[bay.ship.get()] |= type;
+			shipActions[bay.ship.get()] |= type;
+
+	string typeString = ShipEvent::TypeToString(type);
+	auto it = npcActions.find(typeString);
```

Comment:
```suggestion
	auto it = npcActions.find(ShipEvent::TypeToString(type));
```
If it is only used once it is not really worth another variable, is it?